src/core/dbus-manager.c

```diff
@@ -2046,15 +2066,17 @@ static int reply_unit_file_changes_and_free(
 static int method_enable_unit_files_generic(
                 sd_bus_message *message,
                 Manager *m,
-                int (*call)(UnitFileScope scope, UnitFileFlags flags, const char *root_dir, char *files[], UnitFileChange **changes, size_t *n_changes),
+                int (*call)(UnitFileScope scope, UnitFileFlags flags, const char *root_dir, char *files[], UnitFileChange **changes, size_t *n_changes, int (*mac_check)(void *blob, const char *path), void *blob),
```

Comment:
Let's add a typedef for the callback, to keep this definition a bit easier to read.